tensorflow/lite/delegates/nnapi/java/src/main/java/org/tensorflow/lite/nnapi/NnApiDelegate.java

```diff
@@ -0,0 +1,36 @@
+/* Copyright 2019 The TensorFlow Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+==============================================================================*/
+
+package org.tensorflow.lite.nnapi;
+
+import org.tensorflow.lite.Delegate;
+import org.tensorflow.lite.Tensor;
+
+/** {@link Delegate} for NNAPI inference. */
+public class NnApiDelegate implements Delegate {
+
+  private long delegateHandle;
+
+  public NnApiDelegate() {
+    delegateHandle = createDelegate();
+  }
+
+  @Override
+  public long getNativeHandle() {
+    return delegateHandle;
+  }
+
+  private static native long createDelegate();
```

Comment:
We should probably add a close() method. In the future, we may make the delegate stateful, so it'd be good to get into that practice.

Reply:
added a quite simple close() for now